### ISSUE TITLE
Modular usage

### ISSUE THREAD

**Issue by shrpne** [nathanreyes/v-calendar#60]
I want to use only those parts of v-calendar what I really need.
For example, if I need only calendar without date-picker and day-popovers, I don't want to load js and styles of unused components.

Would be great if you make it possible in some of the future releases. 
BTW thanks for this great package, it is the best among vue calendars!

**Comment by nathanreyes** [comment]
I have considered splitting the packages, with `v-calendar` as a dependency for a potential `v-date-picker` plugin. And perhaps even breakout out `v-popover` into its own package.

Until now, it has been very convenient to build them out together in a single package, but as the project stabilizes, who knows.

**Comment by nathanreyes** [comment]
Not sure on how to accomplish this yet. Could include the calendar and picker by default, but allow for importing from `v-calendar/core` to just pull in the calendar. Or could set it up to only pull in the calendar by default and force importing from `v-calendar/picker` to get the date picker. Open for suggestions here.

**Comment by EricMcRay** [comment]
I vote for splitting package. This way package size much more less for anyone who want to use only calendar and date picker over this calendar package is also better that using different datepicker package.

It would be better for development too. While changing some thing for calendar there is a possibility to break date picker and vise versa. I believe decoupling things brings more flexibility and stability. I know it also brings some maintaining difficulties but "great power comes with great responsibility" 😄

**Comment by shrpne** [comment]
I think it's better to keep everything in one package.

> Could include the calendar and picker by default, but allow for importing from v-calendar/core to just pull in the calendar. 

I like this approach, It will be backward compatible with the previous implementation

**Comment by EricMcRay** [comment]
@shrpne is this #105 what you want? If It is, you can actually use components from VCalendar object but there is some bug right now. some details at #112 

Possible usage with current structure:
```
import VCalendar from 'v-calendar'

new Vue({
  el: '#app'
  components: {
    'v-calendar': VCalendar.Calendar,
    'v-datepicker': VCalendar.DatePicker,
   'v-popover': VCalendar.Popover
  }
});
```

**Comment by nathanreyes** [comment]
Yep. This is next on the list. One drawback to the import approach, I believe, is that the CSS is still all bundled together. Still, there isn't an unusually large amount of CSS just for the picker and popover.

**Comment by nathanreyes** [comment]
This issue will get closed with #112.

**Comment by nathanreyes** [comment]
Resolved by 19b8d6faba34fb321a3cfcad226fb863805ba3c5.
